src/native/eventpipe/ep-buffer-manager.c

```diff
@@ -415,7 +453,39 @@ buffer_manager_allocate_buffer_for_thread (
 	EventPipeBuffer *new_buffer = NULL;
 	EventPipeBufferList *thread_buffer_list = NULL;
 	EventPipeSequencePoint* sequence_point = NULL;
-	bool allocate_new_buffer = false;
+	uint32_t sequence_number = 0;
+
+	// Pick a buffer size by multiplying the base buffer size by the number of buffers already allocated for this thread.
+	uint32_t size_multiplier = ep_thread_session_state_get_buffer_count_estimate(thread_session_state) + 1;
+	EP_ASSERT(size_multiplier > 0);
+
+	// Pick the base buffer size based.  Checked builds have a smaller size to stress the allocate/steal path more.
```

Comment:
I think this comment is outdated since we got rid of the "steal" path a while back?

Reply:
```suggestion
	// Pick the base buffer size.  Checked builds have a smaller size to stress the allocate path more.
```